app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -38,7 +42,12 @@ class SecureMessageController @Inject()(
   def createConversation(client: String, conversationId: String): Action[JsValue] = Action.async(parse.json) {
     implicit request =>
       withJsonBody[ConversationRequest] { conversationRequest =>
-        secureMessageService.createConversation(conversationRequest, client, conversationId)
+        secureMessageService.createConversation(conversationRequest.asConversation(client, conversationId)).map {
+          case Right(_)                          => Created
+          case Left(ee: EmailException)          => Created(ee.message)
+          case Left(se: StoreException)          => Conflict(se.message)
```

Comment:
Here we need to match on specialised DuplicateConversationException, as there might be other store exceptions which should not be mapped to Conflict http response code.